app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  # method that returns the 5 most recent comments for a given comment
+  scope :five_most_recent_posts_for_comment, lambda { |post_id|
+                                               where(id: post_id).order(created_at: :desc).limit(5)
```

Comment:
Just as I have stated previously kindly use methods and not scopes as required 😉. You can make use of this simple method instead😉. 

```suggestion
    def five_most_recent_posts_for_comment
      posts.order('created_at Desc').limit(3)
    end
```